herddb-core/src/main/java/herddb/server/ServerConfiguration.java

```diff
@@ -213,6 +213,13 @@
     public static final int PROPERTY_PORT_AUTODISCOVERY = 0;
     public static final int PROPERTY_ZOOKEEPER_SESSIONTIMEOUT_DEFAULT = 40000;
 
+    /**
+     * For HerdDB Collections users most of the time row-level locks are only a waste of CPU/memory
+     * as most of the time one single table is accessed by the only single thread that created it.
+     */
+    public static final String PROPERTY_TABLEMANAGER_CONCURRENT_ACCESS = "server.tablemanager.concurrentaccess";
```

Comment:
It is better to name this property:
PROPERTY_TABLEMANAGER_DISABLE_ROWLEVELLOCKS = "server.tablemanager.disablerowlevellocks"

@diegosalvi  @amitvc  @hamadodene  it would be useful in the future to work this way on the "Follower" because on Follower nodes there is no concurrent access to the same record